dojo/tools/api_sonarqube/api_client.py

```diff
@@ -38,9 +38,8 @@ def __init__(self, tool_config):
                 entry in supported_issue_types for entry in split_issue_types
             )
             if not all_clean:
-                raise Exception(
-                    f"Deteced unsupported issue type! Supported types are {', '.join(supported_issue_types)}"
-                )
+                msg = f"Deteced unsupported issue type! Supported types are {', '.join(supported_issue_types)}"
```

Comment:
While we're modifying these messages, might as well clean up typos...

```suggestion
                msg = f"Detected unsupported issue type! Supported types are {', '.join(supported_issue_types)}"
```